tests/ats/test_app_when_flux_upgraded.py

```diff
@@ -0,0 +1,197 @@
+import os
+from os.path import exists
+from typing import Dict
+
+import pytest
+from pykube import Deployment
+from pytest_helm_charts.clusters import Cluster
+from pytest_helm_charts.flux.git_repository import (
+    make_git_repository_obj,
+    wait_for_git_repositories_to_be_ready,
+    GitRepositoryCR,
+)
+from pytest_helm_charts.flux.kustomization import (
```

Comment:
I didn't notice this before:
Doesn't this mean, that when we upgrade `flux-app` and there is a new `apiversion` of `kustomization`, we also need to release a new version of `pytest_helm_charts`?

Reply:
Yes, like with every other (C)RD. But by keeping the CRD there we make it easy for everyone to write their own tests that use Flux CRs. If the frequency of changes becomes a problem, we can easily split the flux module of pytest-helm-charts into separate sub-plugin - fortunately pytest's plugins are composable.